packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -403,6 +403,6 @@ class RenderEditable extends RenderBox {
     return '$prefix \u2558\u2550\u2566\u2550\u2550 text \u2550\u2550\u2550\n'
            '${text.toString("$prefix   \u2551 ")}' // TextSpan includes a newline
            '$prefix   \u255A\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\n'
-           '$prefix\n';
+           '${prefix.trimRight()}\n';
```

Comment:
I wasted at least 15 minutes learning that we had this code in RenderEditable at all (having first done RenderParagraph)...